sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/MicroBatchExecution.scala

```diff
@@ -47,6 +47,9 @@ class MicroBatchExecution(
     sparkSession, name, checkpointRoot, analyzedPlan, sink,
     trigger, triggerClock, outputMode, deleteCheckpointOnStop) {
 
+  // Exposed for testing purpose.
+  val options: Map[String, String] = extraOptions
```

Comment:
can we create a `LastWriteOptions` instead of exposing it?

Reply:
Yup